database/downloadimg.js

```diff
@@ -0,0 +1,56 @@
+const request = require('request');
+const fs = require('fs');
+const resizeImg = require('resize-img');
+
+async function download(url, dest) {
+
+  const file = fs.createWriteStream(dest);
+
+  await new Promise((resolve, reject) => {
```

Comment:
You can also consider using Promise.resolve() (bluebird) documentation for download function. I used it to organize my pipeline of work for downloading videos.

Reply:
Cool bean, I will look into this.